shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/plants/data/RemoteDbPlantsDataSource.kt

```diff
@@ -1,19 +1,21 @@
 package com.vml.tutorial.plantshop.plants.data
 
+import com.vml.tutorial.plantshop.core.utils.exts.getCollection
 import com.vml.tutorial.plantshop.plants.domain.Plant
 import com.vml.tutorial.plantshop.plants.domain.PlantsDataSource
 import dev.gitlive.firebase.Firebase
 import dev.gitlive.firebase.firestore.firestore
+import dev.gitlive.firebase.firestore.where
 
 class RemoteDbPlantsDataSource : PlantsDataSource {
     private val collection = Firebase.firestore.collection("plants")
 
     override suspend fun getPlants(): List<Plant>? {
```

Comment:
Could you remove the nullable type modifier here? I think we no longer need that. I will clean up the rest of the code :)

Reply:
Done